### ISSUE TITLE
Does the inference notebook work with Code Llama models?

### ISSUE THREAD

**Issue by jackkwok** [philschmid/huggingface-llama-2-samples#16]
Has anyone tried the new Code Llama models from https://huggingface.co/codellama ?

My understanding is the 7B and 13B code-llama set of models have the same architecture as the 7B and 13B Llama-2 models.  So they should work as a drop in replacement right?

**Comment by philschmid** [comment]
Yes and No. Model architecture is the same but they need a different tokenizer.

**Comment by jackkwok** [comment]
@philschmid : thanks for your response.  Does it mean a new Sagemaker Python SDK with updated Huggingface DLC images that include the new Tokenizer(s) will be necessary to use the inference notebook?

**Comment by philschmid** [comment]
Yes

**Comment by jackkwok** [comment]
@philschmid : any idea when the updated Sagemaker Python SDK will be released?
